app/views/items/index.html.erb

```diff
@@ -4,12 +4,13 @@
 <% else %>
   <h1>All Items</h1>
 <% end %>
+
 <section class="grid-container">
   <% @items.each do |item| %>
     <section class = "grid-item" id= 'item-<%=item.id%>'>
       <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
       <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
-      <img src= <%= item.image %>>
+      <%= link_to(image_tag(item.image), "items/#{item.id}") %>
```

Comment:
Could also use the item_path(item) route helper

Reply:
The image_tag didn't like the path helper for some reason